proto/src/constraints.proto

```diff
@@ -32,10 +32,10 @@ message SummaryConstraintMsg {
   oneof second {
      string second_field = 3;
      double value = 4;
-     SummaryBetweenConstraintMsg between = 5;
```

Comment:
Same as the previous question on the other PR, we already merged a "wrong" PR (mainline contains already changes proto files) so this would be the way it should have been done in the first place, after this we only append things and don't change the previous ones if that makes sense.

Reply:
Thanks for explanation, if you add a comment to any of those PRs where you are reverting that we can skip us commenting on that :-)